### ISSUE TITLE
Termination checker should use qualified names when reporting errors

### ISSUE THREAD

**Issue by GoogleCodeExporter** [agda/agda#1140]
``` agda
-- Andreas, 2014-05-22

module _ where

module M where
  f : Set
  f = f

module N where
  f : Set
  f = f

-- OUTPUT IS:
--
-- Termination checking failed for the following functions:
--   f, f
-- Problematic calls:
--   f (at /home/abel/tmp/bla/Agda/test/bugs/TerminationReportQualified.agda:7,7-8)
--   f (at /home/abel/tmp/bla/Agda/test/bugs/TerminationReportQualified.agda:11,7-8)

-- Expected: Qualified names M.f and N.f
```

Original issue reported on code.google.com by `andreas....@gmail.com` on 22 May 2014 at 12:54

**Comment by andreasabel** [comment]
Fixed in 085da78b8ff66a6db9bb98806138fbaeaf7101dc, 
but does not work well for hierarchical modules.

**Comment by andreasabel** [comment]
Pushing a fix soon.
